script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -92,27 +101,33 @@ Vestiges of the time when he was normal.
 [sha:e7307f7f73537b5f3349684e6089d128f3bf44fc]{
 -- Page 665, Offset 10439.
 -- 中世の暗黒時代に培った騎士としての闘争経験が、全身に巡る呪いを打ち払った。
-His battle experience as a knight, forged in the Dark Ages of antiquity, has routed the curse that bound him hand and foot.
+His battle experience as a knight,
+ forged in the Dark Ages,
+ has pushed aside the curse that bound him hand and foot.
```

Comment:
kinda like `routed`.

Reply:
can't say I agree. "routed" without an explicit direction sounds kinda weird to me

Reply:
I feel like you can rout the enemy without direction, but don't care that much.